impeller/entity/geometry.cc

```diff
@@ -628,6 +638,67 @@ GeometryResult StrokePathGeometry::GetPositionBuffer(
   };
 }
 
+template <typename T>
+std::shared_ptr<DeviceBuffer> CreateHostVisibleDeviceBuffer(
```

Comment:
TODO: we should find a way to use the transients buffer for these, or at least have a more efficient allocation strategy.